### ISSUE TITLE
Alternative loss functions

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/DeepClean#24]
While the `PSDLoss` loss function essentially measures PSDR (or ASDR if `asd=True`), and this might function as a good proxy for DeepClean's ability to improve the sensitivity of astrophysical searches, there might be other loss functions out there which measure this more directly with continuous functions and hence be optimized via gradient descent.

Take for example equation (1) in the [SenseMonitor white paper](https://dcc.ligo.org/public/0027/T030276/000/T030276-00.pdf), which gives the average distance to which an interferometer with a given spectral density could detect a BNS inspiral with SNR > 8. The integral in this equation could be replace by a sum over the relevant frequency bins, with a minus sign in to minimize rather than maximize, and in principle we could optimize this equation directly.

This really asks three questions:
- [ ] Are there other loss functions worth considering that might optimize production metrics more directly?
- [ ] How are we going to compare the performance of models trained using two loss functions in a way that produces conclusive answers as to which we ought to use?
- [ ] Does optimizing these metrics more directly ultimately lead to better performance according to the metric(s) which answer the previous question?
